aptos-move/framework/src/release_bundle.rs

```diff
@@ -207,23 +211,23 @@ impl ReleasePackage {
         emitln!(writer, "use supra_framework::code;\n");
 
         if is_testnet && !is_multi_step {
-            emitln!(writer, "fun main(core_resources: &signer){");
+            emitln!(writer, "fun {}(core_resources: &signer){{", function_name);
```

Comment:
Minor, but you could substitute the variable directly into the string to make it a bit easier to read. Same goes for the other occurrences.
```suggestion
            emitln!(writer, "fun {function_name}(core_resources: &signer){{");
```